manual/release-notes.rst

```diff
@@ -15,6 +15,14 @@ more detail.
 12.0.0: not yet released
   - API: breaking changes
 
+    - The header file ``qpdf/QPDFObject.hh`` now generates an error if
+      included. This is to prevent that includes it from accidentally
+      working because an old version is installed somewhere on the
+      system.
```

Comment:
> This is to prevent that includes it from accidentally working because an old version is installed somewhere on the system.

I think something's wrong with this phrase?

Reply:
> I think something's wrong with this phrase?

Thank you. Always read carefully to make sure you didn't any words out, as they say. (Should be "to prevent code that includes it....". I will fix. Thanks for keeping an eye out and taking the time to report. :-)